k8s/daemonset.yaml

```diff
@@ -29,17 +29,16 @@ spec:
         command: [ "sleep", "infinity" ]
         resources:
           requests:
-            memory: "32Mi"
-            cpu: "100m"
+            memory: "250Mi"
+            cpu: "250m"
```

Comment:
Can we leave the requests small or delete them altogether and just bump the limits?
When there is very little resources left on the node I'd still would like to be able to schedule the container and live off the excess of others.